CHANGELOG.md

```diff
@@ -193,43 +29,11 @@ The following classes, methods, properties and events have been deprecated for a
-- Fixed an issue where the CoordinateConversion.mode property was not honored in the CoordinateConversion widget constructor.
-- Fixed an issue where WMSLayer sublayers were being displayed in reverse order in the LayerList widget.
-- Fixed an issue where PopupTemplate content defined with a function was not able to access the feature's geometry even though returnGeometry was set to true.
-- Fixed an issue where the [Legend](/api-reference/esri-widgets-Legend.html) was missing the field alias for field-based heatmap symbology.
-- Fixed an issue where the MapView.hitTest was failing on individual features configured with feature reduction and a large cluster radius.
-- ENH-000085856: Enhanced KMLLayer so it can now display large extent image overlays properly.
-- ENH-000116507 - Enhanced MapImageLayer to highlight the selected feature when displaying its Popup.
-- ENH-000127792: Enhanced the Directions widget with a unit property to easily support different distance units.
-- ENH-000139599: Enhanced KMLLayer to project raster layers as KML files using GCS WGS84 as appropriate.
-- ENH-000146060: Enhanced FeatureReductionCluster to work in any projection supported in the JS API.
-- ENH-000149214: Enhanced View to throw an error in the console if an invalid ID for the view's HTML element is set in the container property.
-- ArcGIS Ideas - 940733: Attachment content file names are automatically hidden when the displayType is preview in Popups.
-- Enhanced the behavior of MapNotesLayer.listMode to have a default value of hide-children instead of show.
-- Added support for feature collections with type markup using MapNotesLayer.
-- Updated CustomContent documentation to allow an optional `PopupTemplateCreatorEvent` parameter for the `PopupTemplateContentCreator` and the `PopupTemplateContentDesroyer` to reference the graphic used to represent the feature.
-- Updated RouteLayer so the resulting feature collection will use ObjectID as the object ID field when saved in a webmap or to a new or existing portal item. Previously the field was named __OBJECTID.
-- Better handling of errors when a layer fails to load.
-- When embedding a video in a popup using the HTML `<video>` tag, the video will automatically resize to fit inside the popup if it is larger than the popup window.
+- TBD
 
 ## Deprecations
 
-The following are deprecated and will be removed in a future release. For anything deprecated in 4.24 and earlier, additional information and links are in the [release notes](https://developers.arcgis.com/javascript/latest/release-notes/#deprecated-classes-properties-methods-events).
+The following are deprecated and will be removed in a future release. For anything deprecated in 4.25 and earlier, additional information and links are in the [release notes](https://developers.arcgis.com/javascript/latest/release-notes/#deprecated-classes-properties-methods-events).
 
 <details>
   <summary>Click to expand!</summary>  
```

Comment:
should we update the list of deprecations too? or are there no changes here yet

Reply:
I can update what's on here with what's on the release notes for 4.25. Is that what we typically do?

Reply:
Thanks @annelfitz ! Updated the list.